test/models/wager_test.exs

```diff
@@ -3,7 +3,7 @@ defmodule MovieWagerApi.WagerTest do
 
   alias MovieWagerApi.Wager
 
-  @valid_attrs %{amount: 42, place: 42}
+  @valid_attrs %{amount: 42, place: 42, user_id: 50, movie_round_id: 55}
```

Comment:
I don't believe you need to make any changes to use `params_with_assocs` as the ModelCase file is [importing the factory.ex file](https://github.com/sbatson5/movie_wager_api/blob/master/test/support/model_case.ex#L25), and the factory.ex file calls `use ExMachina.Ecto`, making `params_with_assocs` available within your tests.